force-app/main/default/classes/HelloWorld.cls

```diff
@@ -0,0 +1,6 @@
+public class HelloWorld {
+    public static String sayHello(String name) {
+        System.debug('### name: ' + name);
```

Comment:
Please, remove System.debug since it's a bad practice.

Well done. Your code is great.